spawner/run.py

```diff
@@ -32,9 +28,10 @@ def main():
     sys.stdout.flush()
     while True:
         try:
-            msg = bus_service.receive_queue_message(queueConf['queue_name'], peek_lock=False)
-            if msg.body is not None:
-                work = msg.body.decode("utf-8")
+            receiver = servicebus_client.get_queue_receiver(queueConf['queue_name'], receive_mode=ServiceBusReceiveMode.RECEIVE_AND_DELETE)
+            received_msgs = receiver.receive_messages()
+            for msg in received_msgs:
+                work = str(b''.join(msg.body), encoding='utf-8')
```

Comment:
since you're sending a dict ValueData in the webserver/run.py, I recommend you using the sample code [here](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/servicebus/azure-servicebus/samples/sync_samples/send_and_receive_amqp_annotated_message.py#L66-L80) to parse message body.

or as mentioned before, you could choose to **jsonfy your dict into a json string**, so that you do not need to handle complicated parsing logic here which is easier for sample code -- which I recommend.

Reply:
@yunhaoling I have updated by your comments.